src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -148,7 +145,7 @@ class ArrayField extends React.Component {
     });
     this.setState(newState, () => {
       this.props.setData(_.set(path, this.state.items, formData));
-      this.scrollToTop();
+      this.scrollToTop(() => focusElement(`.${fieldName} .add-btn`));
```

Comment:
What if the `focusElement` call was a separate statement?
```js
this.scrollToTop();
focusElement(`.${fieldName} .add-btn`);
```
It would focus before it finished scrolling, but is that a problem?

Reply:
It's within the `setState` function, so we need to delay the focus a little to allow the DOM to update before focusing. That's why I added the `focusElement` inside a `setTimeout` within the scrollToTop function - to delay as much as possible.